src/neo4j_genai/experimental/pipeline/pipeline.py

```diff
@@ -130,6 +133,8 @@ async def execute(self, **kwargs: Any) -> RunResult | None:
             status=self.status,
             result=component_result,
         )
+        end_time = default_timer()
+        logger.info(f"Component {self.name} finished in {end_time - start_time}s")
```

Comment:
Do we want this at the `logger.debug` level instead? I'm not sure if it is necessary at the `logger.info` level

Reply:
My understanding is that debug to give developers more fine-grained/detailed info e.g. the every step in making a request to an API and info is higher level and broader e.g. when each API request is made. So info seemed like an appropriate level for timing. In addition you get swamped with logs from all of the dependencies (huggingface, requests, etc.) at debug level so it's very hard to see these messages.

Reply:
I think this granularity of time profiling of different components is not necessary at the info level i.e. if everything is working fine. It's useful if developers want to profile their code and understand the performance o feach different components imo